modules/apps/commerce/commerce-cart-content-web/src/main/resources/META-INF/resources/cart_mini/view.jsp

```diff
@@ -75,12 +75,11 @@ request.setAttribute("view.jsp-portletURL", portletURL);
 				</h4>
 			</div>
 
-			<div class="autofit-col">
-				<liferay-commerce:order-transitions
-					commerceOrderId="<%= commerceCartContentMiniDisplayContext.getCommerceOrderId() %>"
-					cssClass="btn commerce-btn"
-				/>
-			</div>
+			<c:if test="<%= commerceCartContentMiniDisplayContext.isValidCommerceOrder() %>">
+				<div class="autofit-col">
+					<div><a class="btn commerce-btn" href="<%= commerceCartContentMiniDisplayContext.getCommerceCheckoutPortletURL() %>"><liferay-ui:message key="checkout" /></a></div>
```

Comment:
We can't remove the order-transitions tag, because, if an approval workflow is defined on the order, the tag will return the submit button instead of the checkout one.